packages/bap-std/bap-std.2.3.0/opam

```diff
@@ -0,0 +1,79 @@
+opam-version: "2.0"
+name: "bap-std"
+version: "2.3.0"
+maintainer: "Ivan Gotovchits <ivg@ieee.org>"
+authors: "BAP Team"
+homepage: "https://github.com/BinaryAnalysisPlatform/bap/"
+bug-reports: "https://github.com/BinaryAnalysisPlatform/bap/issues"
+dev-repo: "git://github.com/BinaryAnalysisPlatform/bap/"
+license: "MIT"
+build: [
+  ["./configure"
+                 "--prefix=%{prefix}%"
+                 "--with-cxx=`which clang++`"
+                 "--mandir=%{man}%"
+                 "--enable-bap-std"]
+  [make]
+]
+
+install: [
+  [make "reinstall"]
+]
+
+remove: [
+  ["ocamlfind" "remove" "bap"]
+  ["rm" "-f" "%{bin}%/baptop"]
+  ["rm" "-f" "%{bin}%/ppx-bap"]
+]
+
+depends: [
+  "ocaml" {>= "4.08.0" }
+  "base-unix"
+  "bap-future" {= "2.3.0"}
+  "bap-knowledge" {= "2.3.0"}
+  "bitvec" {= "2.3.0"}
+  "bitvec-order" {= "2.3.0"}
+  "bap-core-theory" {= "2.3.0"}
+  "bap-bundle" {= "2.3.0"}
+  "bap-main" {= "2.3.0"}
+  "bap-plugins" {= "2.3.0"}
+  "bap-relation" {= "2.3.0"}
+  "camlzip" {>= "1.07"}
+  "core_kernel" {>= "v0.14" & < "v0.15"}
+  "bin_prot" {>= "v0.14" & < "v0.15"}
+  "fileutils"
+  "graphlib" {= "2.3.0"}
+  "oasis" {build & >= "0.4.7"}
+  "ocamlfind" {>= "1.5.6" & < "2.0"}
+  "ppx_bap" {>= "v0.14" & < "v0.15"}
+  "regular" {= "2.3.0"}
+  "uri"
+  "utop" {build & >= "2.0.0"}
+  "uuidm"
+  "zarith"
+  "cmdliner" {>= "0.9.8"}
+  "ogre" {= "2.3.0"}
+  "monads" {= "2.3.0"}
+  "result"
+]
+depexts: [
+  ["libgmp-dev" "libzip-dev" "clang"] {os-distribution = "ubuntu"}
+  ["gmp" "libzip"] {os = "macos" & os-distribution = "macports"}
+  ["clang"] {os-distribution = "debian"}
+  ["binutils" "gmp-dev" "libzip-dev" "m4" "perl" "zlib-dev"] {os-distribution = "alpine"}
+
+
+]
```

Comment:
```suggestion
  "conf-gmp" {build}
  "conf-zlib" {build}
  "conf-clang" {build}
  "conf-which" {build}
]
```
This would increase the maintainability of this package and would make several other (and future) distributions work out-of-the-box